docs: initial list of related technologies of Keptn (#795)
keptn/lifecycle-toolkit: CONTRIBUTING.md

# Contributing

## Ancillary Technologies

> This section serves as a reference for various supplementary technologies associated with Keptn. 
> The information has been gathered from the community and is subject to alteration. 
> It is suggested that this list be utilized as a means of gaining familiarity with the ancillary technologies of Keptn. 
> If you believe that additional content should be included in this list, please submit a comment on issue
> [#733](https://github.com/keptn/lifecycle-toolkit/issues/733).

Reviewer: ```suggestion
You should understand some ancillary technologies
to effectively use and contribute to the Keptn lifecycle-toolkit.
This section provides links to some materials that can help your learning.

The information has been gathered from the community and is subject to alteration. 

If you have suggestions about additional content that should be included in this list, please submit a comment on issue
[#733](https://github.com/keptn/lifecycle-toolkit/issues/733).
```

> [#733](https://github.com/keptn/lifecycle-toolkit/issues/733).

Reviewer: Better to tell them to file an issue.  #733 will be closed in a few weeks.

Author: updated accordingly

<details>

>   <summary>Kubernetes</summary>

Reviewer: Should you include a link to the Kubernetes docs?

Author: yes, that's correct. We should add a link to the official documentation

Author: added

- **Understand the basics of Kubernetes**
  - [ ]  [Kubernetes official documentation](https://kubernetes.io/docs/concepts/overview/)
  - [ ]  [Kubernetes For Beginner](https://youtu.be/X48VuDVv0do)
- **Kubernetes Architecture**
  - [ ]  [Philosophy](https://youtu.be/ZuIQurh_kDk)
  - [ ]  [Kubernetes Deconstructed: Understanding Kubernetes by Breaking It Down](https://www.youtube.com/watch?v=90kZRyPcRZw)
- **CRD**
  - [ ]  [Custom Resouce Definition (CRD)](https://www.youtube.com/watch?v=xGafiZEX0YA)
  - [ ]  [Kubernetes Operator simply explained in 10 mins](https://www.youtube.com/watch?v=ha3LjlD6g7g)
  - [ ]  [Writing Kubernetes Controllers for CRDs](https://www.youtube.com/watch?v=7wdUa4Ulwxg)
- **Kube-builder Tutorial**
  - [ ] [book.kubebuilder.io](https://book.kubebuilder.io/introduction.html)
- **Isitobservable**
  - [ ] Keptn has tight integrations with Observability tools and therefore knowing how to _Observe a System_ is important.
  - [ ] [Isitobservable website](https://isitobservable.io/)
  - [ ] [Is it Observable? with Henrik Rexed](https://www.youtube.com/watch?v=aMwk2qo0v40)
</details>

<details>
  <summary>Understanding SLO, SLA, SLIs</summary>

- **Overview**

>   - [ ]  [overview](https://www.youtube.com/watch?v=tEylFyxbDLE)

Reviewer 2: nit for consistency
```suggestion
  - [ ]  [Overview](https://www.youtube.com/watch?v=tEylFyxbDLE)
```

Author: updated

## Prerequisites

> ## Ancillary Technologies

Reviewer 3: ```suggestion
## Related Technologies
```

> You should understand some ancillary technologies

Reviewer 3: ```suggestion
You should understand some related technologies
```

Author: updated
